### PR TITLE
COMP: correct CMake parameters (generator name, code folder)

### PR DESCRIPTION
**Pull request by mauigna06**
Just like it was asked [here](https://discourse.slicer.org/t/building-slicer-from-source/17823/2). 
I made a pull request to set up the lines I used to build Slicer and correct the old cmake line that appears on the developer build tutorial for Windows.

I didn't test all the modules after executing the compiled Slicer, just that the sliceViews and the 3D view worked. I did't build debug version, only release

### CODE REVIEW

File: Docs/developer_guide/build_instructions/windows.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ Release mode:
 ```
 mkdir C:\D\S4R
 cd /d C:\D\S4R
-"C:\Program Files\CMake\bin\cmake.exe" -G "Visual Studio 16 2019 Win64" -DQt5_DIR:PATH=C:\Qt\5.15.0\msvc2019_64\lib\cmake\Qt5 C:\D\S4
+"C:\Program Files\CMake\bin\cmake.exe" -G "Visual Studio 16 2019" -A x64 -DQt5_DIR:PATH=C:\Qt\5.15.2\msvc2019_64\lib\cmake\Qt5 C:\D\S4\Slicer
```
Comment (lassoan) [inline Docs/developer_guide/build_instructions/windows.md:44]:
```suggestion
"C:\Program Files\CMake\bin\cmake.exe" -G "Visual Studio 16 2019" -A x64 -DQt5_DIR:PATH=C:\Qt\5.15.2\msvc2019_64\lib\cmake\Qt5 C:\D\S4
```

File: Docs/developer_guide/build_instructions/windows.md [hunk 2]
```diff
@@ -50,7 +50,7 @@ Debug mode:
 ```
 mkdir C:\D\S4D
 cd /d C:\D\S4D
-"C:\Program Files\CMake\bin\cmake.exe" -G "Visual Studio 16 2019 Win64" -DQt5_DIR:PATH=C:\Qt\5.15.0\msvc2019_64\lib\cmake\Qt5 C:\D\S4
+"C:\Program Files\CMake\bin\cmake.exe" -G "Visual Studio 16 2019" -A x64 -DQt5_DIR:PATH=C:\Qt\5.15.2\msvc2019_64\lib\cmake\Qt5 C:\D\S4\Slicer
```
Comment (lassoan) [inline Docs/developer_guide/build_instructions/windows.md:53]:
```suggestion
"C:\Program Files\CMake\bin\cmake.exe" -G "Visual Studio 16 2019" -A x64 -DQt5_DIR:PATH=C:\Qt\5.15.2\msvc2019_64\lib\cmake\Qt5 C:\D\S4
```

**Review by lassoan — COMMENTED** [review]
Thank you, this is useful. I've added two comments to keep `C:\D\S4` as source folder, because that's what is described in the build instructions. If this was not clear in the documentation then then maybe you could suggest changes that would make this less ambiguous.

**Comment by mauigna06** [comment]
The tutorial is okey I just missed a the point on the git clone command because it is on the line below

**Comment by lassoan** [comment]
Thank you! Merged.

**Pull request merged**